Automattic/jetpack
# Search: Update and apply ContextualUpgradeTrigger to Jetpack Search dashboard

Fixes #26599 

#### Changes proposed in this Pull Request:

* Update `ContextualUpgradeTrigger` component styles in line with the design of Jetpack Library.

<img width="1294" alt="截圖 2022-10-05 下午4 42 50" src="https://user-images.githubusercontent.com/6869813/194019385-3e298e6e-5ebf-4df9-99c4-420cb1073084.png">

* Replace the upgrade trigger section of the Jetpack Search dashboard with the `ContextualUpgradeTrigger` component.
* Apply CTA of checkout to `ContextualUpgradeTrigger`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


N/A

#### Testing instructions:






* Prepare a site with `Jetpack` connected and a `Jetpack Seach` subscription.
* Navigate to the site with URL `/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`.
* Ensure the upgrade trigger section looks the same as previously.
* Click on the upgrade trigger section.
* Ensure the page will be loading and redirected outside to the Jetpack connection page.

> Note: Since the new pricing tier is still in progress, the checkout flow would fail unless the Jetpack Search subscription is removed before clicking on the upgrade trigger.
